docs/examples/bci2000/connect.md

```diff
@@ -0,0 +1,133 @@
+# Connecting DSI Headset
+
+This guide will walk you through how to connect your Wearable Sensing DSI headset to BCI2000 for data acquisition.
+
+```{admonition} Note
+:class: sidebar note
+Please ensure BCI2000 is already installed on your system. If you haven't installed it yet, you can find the download instructions [here](https://www.bci2000.org/mediawiki/index.php/DownloadBCI2000).
+```
+
+## Update the DSI API Driver
```

Comment:
We determined that the update was not necessary. I would say configure the BCI2000:DSISerial module and modify the flow below.